theme/static/css/screen.css

```diff
@@ -151,7 +150,7 @@ blockquote {
 }
 
 .page {
-    margin: 0 90px;
+    margin: 0 0px;
```

Comment:
It's better to use `margin: 0;`, although 0 is probably the default so you might be able to remove the entire `.page` block.

Reply:
Hmm. I think I'l leave the `.page` block, since it makes it more obvious what the defaults are and what get's overridden. I mean it makes it easier to just search for `.page` in the document to see the different styles used for different screen sizes.